README.md

```diff
@@ -129,6 +132,16 @@ Popovers also benefit from a `hide` API made publically acessible:
 {{/popover-on-element}}
 ```
 
+## Targets
```

Comment:
I have added this section of the documentation to describe what a target is in the context of tooltips and popovers. This should make the rest of the documentation less verbose (we can just say 'target' and link back to this section for more info).